### PR TITLE
CEREBRAL-4191 Delete code

### PR DESCRIPTION
**Pull request by dustinhiatt-wf**
We only need this repo to maintain the helm chart for redirect. Kill everything else.

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
<details>
<summary>(3) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>activesupport < 5.2.4.3</code> via <code>Gemfile.lock</code></li>
<li>1 vulns in <code>kramdown < 2.3.0</code> via <code>Gemfile.lock</code></li>
<li>2 vulns in <code>rack < 2.1.4</code> via <code>Gemfile.lock</code></li>
</details>
<details>
<summary>(1) Image validation issues were detected</summary>
<li>Repo does not appear to install package updates in deployable image</li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=slate">this Splunk dashboard</a> for more CVE details
* Ensure <a href="https://wiki.atl.workiva.net/display/SECURITY/Security+Requirements+for+Building+Containers#SecurityRequirementsforBuildingContainers-PackageUpdating">package updates</a> are being installed in deployed image
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: Dockerfile [hunk 1]
```diff
@@ -1,103 +1,7 @@
-FROM debian:stretch-slim as helm_artifact
```
Comment (andrewunderwood-wf) [inline Dockerfile:1]: You still need the helm stage because you still need to build a Helm artifact

File: Dockerfile [hunk 2]
```diff
@@ -1,103 +1,7 @@
...
-RUN http-server --version
-
-# bring the static html in
-COPY --from=builder /local-build/build/ s/cerebral-docs/
-
-# bring in the generated clients
-COPY --from=swagger-builder /local-build/source/generated/ s/cerebral-docs/generated/
+FROM scratch
 
 # run as non-root user
```
Comment (andrewunderwood-wf) [inline Dockerfile:4]: I don't believe the user and CMD statements will do anything on `scratch`, you should be able to just end on `FROM scratch`

File: README.md [hunk 3]
```diff
@@ -34,82 +34,8 @@ Getting started with Slate is super easy! Simply fork this repository and follow
 Getting Started with Slate
 ------------------------------
 
-### Prerequisites
+# NOTICE
```
Comment (andrewunderwood-wf) [inline README.md:37]: Minor nit, but I would remove everything above this in the Readme so you just see this message when you load the repo.

**Review by andrewunderwood-wf — COMMENTED** [review]
From `helm/cerebral-docs/templates/` you should be able to remove everything but:
ingress.yaml

From `helm/cerebral-docs/values.yaml` you should be able to remove everything except the `ingress` and `labels` blocks.

**Review by davidgiametta-wk — APPROVED** [review]

**Comment by dustinhiatt-wf** [comment]
QA +1

Godspeed slate

**Comment by dustinhiatt-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**